sdk/tests/deqp/functional/gles3/es3fFramebufferBlitTests.js

```diff
@@ -726,7 +726,7 @@ goog.scope(function() {
         for (var ndx = 0; ndx < 2; ndx++) {
             ctx.bindFramebuffer(gl.FRAMEBUFFER, ndx ? dstFbo : srcFbo);
             if (ndx) {
-                gradientToDstShader.setGradient(ctx, gradShaderDstID, dstRangeInfo.valueMax, dstRangeInfo.valueMin);
```

Comment:
That's true. But the C++ code have issue too, I think. 
See the callee where it use valueMax and valueMin: https://github.com/KhronosGroup/WebGL/blob/master/sdk/tests/deqp/functional/gles3/es3fFboTestUtil.js#L251. 

And the shader where it use the uniform value: https://github.com/KhronosGroup/WebGL/blob/master/sdk/tests/deqp/functional/gles3/es3fFboTestUtil.js#L235. 

I didn't see why we need to reverse the valueMin and valueMax.